docs/usage-guidelines.md

```diff
@@ -0,0 +1,51 @@
+
+### Project must be of use for others
+While it can be nice to have a CI for your library to upload and share, should you limit it to libraries and projects that can have a use for others.  
+Effectively, if the project's purpose is to ease your coding of other Projects, while not having any use outside of that for others, is it not something you can and should put on our CI.
+
+### Project must be properly maintained
+CodeMC is not meant as a cheap way of storing your project's final jar file without any updates to it. As such are we requiring your project to be actively maintained by receiving regular updates.  
+Do note that we consider the following as maintaining a project:
+
+- Creating commits with proper messages.
+- Keeping dependencies updated.
+- Keeping the project's issue and Pull request tracker clean by closing issues, closing/merging pull requests, labeling them, assigning people, etc.
+
+With all this said are we aware that there can be justifiable reasons for a projects being stale. In such cases do we ask you to inform us about it.
+
+### Project cannot encourage nor support illegal activities or content
+We want the programming community to be as welcoming and inclusive as possible. As such are we not tollerating projects that encourage, include or support hate-speech, racism, sexism, bigotry or any other form of harassment, or illegal content.  
+Should you find such a project on our CI are your obligated to report it to us as soon as possible to be dealt with.
+
+### Project should have an english documentation available
+CodeMC accepts non-english projects. However, to allow a user to easily understand what this project does and is about, should it provide easy access to an english documentation of the Project's README or other noteworthy documentation a user may want to read.
+
+Translations done through a translator without proofreading by an english speaking person should have a note of not being 100% accurate and done via translators.
+
+## Reasonable Usage Limits
+We do not apply usage limits (Rate limits) on most of our CI. For parts where we do this are we giving relatively generous limits.  
+In order to keep this state and not force harsher limits on anyone are we requesting you to not abuse these limits in any way. This means to keep frequency of builds and uploads to the Nexus to a reasonable limit.
+
+## Punishments
+We reserve the right to punish you in the following ways, should you be confirmed to have broken these guidelines:
+
+- Restricting access to our Nexus and CI.
+- Removal of builds and Nexus uploads.
+- Removal of your Project(s).
+- Removal of your account(s).
+- Complete blocking of access to our CI and/or Nexus.
+
+CodeMC may issue one or multiple of the above punishments at any given time at our own discretion.
```

Comment:
I usually like to keep things explicit and clear whenever punishments are involved.
```suggestion
CodeMC may issue one or multiple of the above punishments at any given time at our discretion. Any of these actions may happen in any order of any severity. 
```